l10n_es_account_summary/account_summary.py

```diff
@@ -0,0 +1,91 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    Odoo - Account summary
+#    Copyright (C) 2014 Luis Martinez Ontalba (www.tecnisagra.com).
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as published
+#    by the Free Software Foundation, either version 3 of the License, or
+#    (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+import tools
+from osv import fields,osv
+import decimal_precision as dp
+
+class account_account(osv.osv):
```

Comment:
Corregido

El 05/01/2015 a las 12:01, Pedro M. Baeza escribió:
>
> In l10n_es_account_summary/account_summary.py 
> <https://github.com/OCA/l10n-spain/pull/84#discussion-diff-22455642>:
>
> > +#
> > +#    This program is distributed in the hope that it will be useful,
> > +#    but WITHOUT ANY WARRANTY; without even the implied warranty of
> > +#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
> > +#    GNU Affero General Public License for more details.
> > +#
> > +#    You should have received a copy of the GNU Affero General Public License
> > +#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
> > +#
> > +##############################################################################
> > +
> > +import tools
> > +from osv import fields,osv
> > +import decimal_precision as dp
> > +
> > +class account_account(osv.osv):
>
> Utiliza orm.Model
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/OCA/l10n-spain/pull/84/files#r22455642>.
>